<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSetupDialog</class>
 <widget class="QDialog" name="ExportSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="trustedLabel">
     <property name="text">
      <string>&amp;Trusted:</string>
     </property>
     <property name="buddy">
      <cstring>trustedCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="trustedCheckBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="includeServerLabel">
     <property name="text">
      <string>Include &amp;Server:</string>
     </property>
     <property name="buddy">
      <cstring>includeServerCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="includeServerCheckBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Password</string>
     </property>
     <property name="buddy">
      <cstring>passwordLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="titleLabel"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trustedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trustedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
